### PR TITLE
Storybook 📖

### PR DESCRIPTION
**Pull request by vefimov**
screen-2022-07-26-15-52-54-9jxi8

## What's done 
- added storybook + updated README
- new component `components/Button` - finally there is one component for button, just set props to change the style
- new component `components/Tabs` 
- removed component `components/button`

[Storybook docs](https://storybook.js.org/docs/react/get-started/introduction)

## Documented components
- Accordion
- Dropdown
- Autocomplete
- Buttons
- Avatar List
- Colors (pallete)
- SVGR icons - SVG icons represented as React components 
- SVG icons
- Safe Image
- Table
- Tag
- Typography
- Header buttons
- Tabs

## Video
https://www.loom.com/share/86a5d70530fa4de1a6c03a42283f92b5

### CODE REVIEW

**Review by Lamperoyge — COMMENTED** [review]
awesome! I think there are some conflicts but this is great!

**Review by Lamperoyge — APPROVED** [review]

File: wondrous-app/components/Common/Image/index.tsx [hunk 1]
```diff
@@ -29,8 +47,9 @@ export const SafeImage = (safeImageArgs: SafeImageArgs) => {
     objectPosition = 'center',
     ...props
   } = safeImageArgs;
+
   const [imgUrl, setImageUrl] = useState(null);
-  const hasProtocol = typeof src === 'string' && (src?.startsWith('https') || src?.startsWith('file://'));
+  const hasProtocol = typeof src === 'string' && (src?.startsWith('http') || src?.startsWith('file://'));
```
Comment (andywong418) [inline wondrous-app/components/Common/Image/index.tsx:52]: Think we would only want to accept https

**Review by andywong418 — COMMENTED** [review]
This is awesome! Just out of curiosity, is there any standard practice on what we would turn into a Storybook component in the future? Should design be looped into this? 

Otherwise just a comment from me. Also let's fix merge conflicts

**Comment by vefimov** [comment]
> This is awesome! Just out of curiosity, is there any standard practice on what we would turn into a Storybook component in the future? Should design be looped into this?
> 
> Otherwise just a comment from me. Also let's fix merge conflicts

Ideally, each component should have a Storybook story or set of stories. About the design, we can integrate Storybook with Figma and Figma with Storybook. 

- [Embed Storybook in Figma](https://storybook.js.org/docs/react/sharing/design-integrations#embed-storybook-in-figma-with-the-plugin)
- [Embed Figma in Storybook](https://storybook.js.org/docs/react/sharing/design-integrations#embed-figma-in-storybook-with-the-addon)

File: wondrous-app/components/Common/Image/index.tsx (hunk 1 above)
Comment (vefimov) [reply wondrous-app/components/Common/Image/index.tsx:52]: We always use HTTPS except for localhost, I've changed to the http, so the storybook could load images locally.

**Pull request merged**